### PR TITLE
WP 4.6 Compatibility

### PR DESCRIPTION
**Pull request by ChaseWiseman**

### CODE REVIEW

**Comment by ChaseWiseman** [comment]
@maxrice when you get a chance, can you review these changes? They're pretty minor and take care of the only conflicts I've noticed with the WP 4.6 beta.

File: woocommerce/api/abstract-sv-wc-api-json-request.php [hunk 1]
```diff
@@ -91,7 +91,7 @@ public function get_params() {
 	 */
 	public function to_string() {
 
-		return json_encode( $this->get_params() );
+		return ! empty( $this->get_params() ) ? json_encode( $this->get_params() ) : '';
```
Comment (unfulvio) [inline woocommerce/api/abstract-sv-wc-api-json-request.php]: @ChaseWiseman  is this ` ! empty( $this->get_params() )` valid in PHP 5.2?

Comment (ChaseWiseman) [inline woocommerce/api/abstract-sv-wc-api-json-request.php]: empty strikes again! Good catch @unfulvio

**Comment by maxrice** [comment]
👍  LGTM

**Pull request merged**